Wartości Laboratorium EE v1.1.md

```diff
@@ -0,0 +1,165 @@
+    produktami/usługami - czy to Klient, Współpracownik czy Odbiorca - czerpała z
+    tej interakcji przyjemność.
+
+- **Inkluzywność** - Jesteśmy otwarci na każdego niezależnie od jego/jej
+pochodzenia, wyznania, płci, koloru skóry, poglądów politycznych,
+orientacji seksualnej, wzrostu, wykształcenia czy statusu.
+
+   - **Unikamy treści i działań mogących być uznane za:**   
+    Seksistowskie;  
+    Mobbing;  
+    Dyskryminujące;    
+    Nękanie lub Szykanowanie;
+
+- **Odpowiedzialność** - Mamy świadomość wpływu naszych decyzji na otoczenie,
+rozumiemy że odpowiadamy za ich rezultaty zarówno indywidualnie, jak zespołowo.
+
+- **Innowacyjność** - Dzielimy się swoimi pomysłami, nie boimy się tworzyć.
+Nie ograniczamy się to do tego, co znane i bezpieczne. Szukamy nowych rozwiązań
+i potrafimy wdrożyć najlepsze z nich.
+
+- **Pokora** - Mamy świadomość że na swojej drodze spotykamy osoby bardziej
+doświadczone, mądrzejsze od nas - szanujemy je i uczymy się od nich.
+Nie boimy się przyznać do błędu.
+
+- **Ciekawość** - Chętnie słuchamy innych perspektyw oraz odkrywamy nowe rzeczy
+w zakresie swojej profesji.
+
+### Dobre praktyki, które pomogą w osiągnięciu postawionego celu:
+
+- **Szanować czas i zaangażowania innych** -
+Wszyscy mamy zadania, które musimy zrealizować, pamiętaj o tym prosząc kogoś
+o pomoc czy zlecając dodatkowe zadania.    
+**Pamiętaj o zasadzie "[you must try and then you must
+ask](https://blogs.akamai.com/2013/10/you-must-try-and-then-you-must-ask.html)"**
+
+- **Nie bać się zadawać pytania** - Nawet jeśli ktoś nie może Ci w danym momencie
+pomóc, poproś o zaproponowanie terminu rozmowy, albo o przesłanie linków
+do materiałów do samodzielnej nauki.
+
+- **Dbać o przestrzeń Laboratorium** - Open Space jest dostępny dla wszystkich
+współpracowników. Każdy ma swoje indywidualne potrzeby i oczekiwania względem
+stanowiska pracy, choć odpowiadamy za nią wszyscy.
+
+- **Nie bać się mówić na głos o rzeczach, które sprawiają nam trudności** - Nikt nie zakłada,
+że umiemy/możemy wszystko. Zawsze możesz liczyć na swój zespół.
+
+- **Nie krytykować innych** - zamiast tego dajemy feedback (czyli informację dotyczącą
+efektu pracy, która wskazuje jednocześnie kierunek rozwiązania problemu i uniknięcia
+jego ponownego wystąpienia) lub proponujemy wsparcie.
+
+- **Zwracać uwagę jeśli ktoś zachowuje się niezgodnie z zasadami opisanymi
+w tym dokumencie - zwróć uwagę i odpuść** - pamiętaj też, że wszyscy powinniśmy
+ambasadorami tych zasad! Jeśli to nie poskutkuje możesz poprosić o wsparcie
+Janka [treść rozmów jest traktowana jako poufna, podobnie jak tożsamość osoby
+zgłaszającej]
+
+- [Dodaj swoje propozycje]
+
+## Inne
+
+### Zasady - Bezpieczeństwo
+- **Maile** - zawsze warto zrobić doublecheck odbiorców do których wysyłasz
+wiadomość i mieć ustawioną swoją stopkę Laboratoryjną.
+
+- **Komputery** - odchodząc od komputera zablokować ekran/wyłączyć go.
+
+- **Narzędzia** - pamiętaj żeby aktywować dodatkowe opcje zabezpieczające
```

Comment:
To znaczy, że obowiązkiem jest aktywowanie tych opcji, czy to tylko dobra praktyka? Myślę, że warto jednoznacznie to napisać.